factory-hooks/db-update/site-update.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# Factory Hook: db-update
+#
+# The existence of one or more executable files in the
+# /factory-hooks/db-update directory will prompt them to be run
+# *instead of* the regular database update (drush updatedb) command. So
+# that update command will normally be part of the commands executed
+# below.
+#
+# Usage: SCRIPTNAME site env db-role domain custom-arg1 custom-arg2 ...
+# Map the script inputs to convenient names.
+# Acquia hosting site / environment names
+site="$1"
+env="$2"
+
+# database role. (Not expected to be needed in most hook scripts.)
+db_role="$3"
+
+# The public domain name of the website.
+domain="$4"
+
+# Custom argument passed with the code update UI.
+# Custom argument.
+# custom_argument="$5"
+
+# The websites' document root can be derived from the site/env:
+docroot="/var/www/html/$site.$env/docroot"
+
+# Acquia recommends the following two practices:
+# 1. Hardcode the drush version.
+# 2. When running drush, provide the application + url, rather than relying
+#    on aliases. This can prevent some hard to trace problems.
+DRUSH_CMD="drush8 --root=$docroot --uri=https://$domain"
```

Comment:
Checking drush version - I think we might need drush 9 or drush 10?

Reply:
I changed the version to drush 10. I also confirmed this is available on the Acquia servers.